src/pages/home.tsx

```diff
@@ -1,14 +1,64 @@
-import React from "react";
+import React, { useContext } from "react";
+import { HiArrowTopRightOnSquare, HiUserCircle } from 'react-icons/hi2'
+import { RouteContext } from "../App";
+import OpenSaucedLogo from "../assets/opensauced-logo.svg";
+import { useAuth } from "../hooks/useAuth";
+import { useOpensaucedUserCheck } from "../hooks/useOpensaucedUserCheck";
 
-interface HomeProps {
-  osAccessToken: string;
-  setRenderedPage: (page: string) => void;
-}
+function Home() {
+  const {setCurrentPage} = useContext(RouteContext)
+  const {user} = useAuth()
+  const {currentTabIsOpensaucedUser, checkedUser} = useOpensaucedUserCheck()
 
-function Home({ osAccessToken, setRenderedPage }: HomeProps) {
   return (
-    <div>
-      <p className="text-white">Home</p>
+    <div className="grid grid-cols-1 divide-y divide-white/40 divider-y-center-2 min-w-[320px] text-white">
+      <header className='flex justify-between'>
+        <img src={OpenSaucedLogo} alt="OpenSauced logo" className='w-[45%]' />
+        {user &&
+          <button 
+            onClick={e=> {
+              setCurrentPage('profile', {userName: user.user_name})
+            }}
+            className='flex gap-1 items-center hover:text-orange text-white no-underline'>
+            {user?.user_name}
+            <img
+              src={`https://github.com/${user?.user_name}.png`}
+              alt="profile image"
+              className='rounded-full w-6 aspect-square border border-orange' />
+          </button>}
+      </header>
+      <main className='main-content'>
+        <h3 className='text font-medium text-base leading-10'>Tools:</h3>
+        <div className='tools flex flex-col gap-2'>
+          <a
+            target='_blank'
+            href={`https://insights.opensauced.pizza/feed`}
+            className='flex items-center bg-slate-700 hover:bg-slate-700/70 text-white hover:text-orange no-underline gap-2 p-1.5 px-3 w-full rounded-sm font-medium text-sm'
+          >
+            <HiArrowTopRightOnSquare />
+            Go to Highlight page
+          </a>
+          <a
+            target='_blank'
+            href={`https://insights.opensauced.pizza/user/${user?.user_name}`}
```

Comment:
```suggestion
            href={`https://insights.opensauced.pizza`}
```
This should go to dashboard and not profile page.